docs/getting-started.md

```diff
@@ -205,6 +205,15 @@ Now that you're setup, if you can run it on the command line, you can run it in
             - To start normal gameplay from Kickoff, press "Stop", then "Kickoff" for either team and then "Normal Start".
             - To learn more about how we coordinate different RefereeCommands to start special case gameplay behaviour (PenaltyKick, CornerKick, FreeKick), look at the [SSL rule documentation](https://ssl.robocup.org/rules/).
         - In addition, you can use ctrl-click to move the ball around in the Simulator, or try changing the Play Override on the Visualizer to select specific Plays!
+    
```

Comment:
the getting_started.md is also tagged with #2711 
could you update the guide with this?